### PR TITLE
Parser for logfile var log up2date 1.x

### PR DESCRIPTION
**Pull request by TripleJQK**
@xiangce need review on new parser fro /var/log/up2date log file

### CODE REVIEW

File: insights/config/specs.py [hunk 1]
```diff
@@ -437,6 +437,7 @@
     "uname"                     : First([CommandSpec("/bin/uname -a"),
                                     SimpleFileSpec("uname")]),
     "up2date"                   : SimpleFileSpec("etc/sysconfig/rhn/up2date"),
+    "up2date.log"               : SimpleFileSpec("var/log/up2date", large_content=True),
```
Comment (xiangce) [inline insights/config/specs.py]: Better use the same spec name in 1.x and 3.x for the newly added spec.

File: insights/parsers/up2date_log.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from .. import LogFileOutput, parser
+
+
+@parser('up2date.log')
+class Up2dateLog(LogFileOutput):
```
Comment (xiangce) [inline insights/parsers/up2date_log.py]: `Up2DateLog`?

File: insights/parsers/up2date_log.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    """Class for parsing the log file: ``/var/log/up2date`` """
+    pass
```
Comment (xiangce) [inline insights/parsers/up2date_log.py]: The new sub-classes inherits from LogFileOutput, you should check if the default `time_format` can cover the actual time format of the new log-file. The default time format is 

`time_format = '%Y-%m-%d %H:%M:%S'`

It does not match with the time stamp string in up2date. Please modify it correspondingly and use `get_after` in test to verify it.

File: insights/parsers/up2date_log.py (hunk 2 above)
Comment (TripleJQK) [reply insights/parsers/up2date_log.py]: the log file name is /var/log/up2date,  as the up2date is one word, so Up2dateLog

File: insights/config/specs.py (hunk 1 above)
Comment (TripleJQK) [reply insights/config/specs.py]: I check the parser for rhsm.log in both 1.x and 3.x:
In 1.x, the spec is "rhsm.log"
In 3.x, the spec is rhsm_log with alias rhsm.log(using Specs.rhsm_log in parser file. If rhsm.log, Specs.rhsm.log seems a little werid).

So use up2date_log in both 1.x and 3.x will be better?

File: insights/parsers/up2date_log.py (hunk 3 above)
Comment (TripleJQK) [reply insights/parsers/up2date_log.py]: I will modify this

File: insights/config/specs.py (hunk 1 above)
Comment (TripleJQK) [reply insights/config/specs.py]: use up2date_log instead of up2date.log as same as 3.x

File: insights/parsers/up2date_log.py (hunk 2 above)
Comment (TripleJQK) [reply insights/parsers/up2date_log.py]: not change this as my last comment

File: insights/parsers/up2date_log.py (hunk 3 above)
Comment (TripleJQK) [reply insights/parsers/up2date_log.py]: done

**Comment by xiangce** [comment]
See comments: https://github.com/RedHatInsights/insights-core/pull/920#discussion_r165290734

**Comment by TripleJQK** [comment]
@xiangce modified as your request, please review when free

**Pull request merged**